docs/source/existing_code.rst

```diff
@@ -0,0 +1,174 @@
+.. _existing-code:
+
+Using mypy with an existing codebase
+====================================
+
+This section explains how to get started using with an existing,
+significant codebase that has no or few type annotations. If you are
```

Comment:
"no or few" -> "little to no".

Also, maybe add a blurb saying something about how this page is useful specifically for codebases where it's difficult to add type hints all at once -- I think that would do a better job of explaining what we mean by a "significant codebase" and make it more clear why the page is recommending an incremental approach.